documentation/internal/Filters-for-Applications.md

```diff
@@ -134,83 +136,108 @@ Options:
 
-<a name="action"></a>
+- `pending` or `PENDING`
+- `approved` or `APPROVED`
+- `rejected` or `REJECTED`
 
-#### action
+<a name="applicant-action"></a>
 
-Example: `action=edit-draft`  
-Considered in implementation: **Yes** [#251 - Filters]  
-Can have combined values: **Yes** e.g: `action=edit-draft,make-updates`  
-Options:
+### applicant action _not implemented_
 
 - Applicant:
   `edit-draft`
   `make-updates`
   `renew`
   `view` (Submitted)
-- Reviewer
-  `start-review`
-  `continue-review`
-  `review-updates`
-  `view` (Submitted)
-- Consolidator
-  `assign`
-  `consolidate`
-  `review-updates`
-  `view` (Submitted)
+
+<a name="assigner-action"></a>
+
+#### assigner action
+
+Check out in schema `AssignerAction` ENUM and Function `assigner_list`
+Included: **Yes**
+Can have combined values: **No**
+Examples: `assigner-action=ASSIGN` or `assigner-action=RE_ASSIGN`
+After is converted to Graphql: `assignerAction: {equalTo: "RE_ASSIGN"}`
+Options:
+
+- `ASSIGN`
+- `RE_ASSIGN`
+
+<a name="reviewer-action"></a>
+
+#### reviewer action
+
+Check out in schema `ReviewerAction` ENUM and Function `review_list`
+Included: **Yes**
+Can have combined values: **No**
+Examples: `reviewer-action=start_review` or `reviewer-action=CONTINUE_REVIEW`
+After is converted to Graphql: `reviewerAction: {equalTo: "CONTINUE_REVIEW"}`
+Options:
+
+- `SELF_ASSIGN`
+- `START_REVIEW`
+- `CONTINUE_REVIEW`
+- `RESTART_REVIEW`
+- `UPDATE_REVIEW`
+- `VIEW_REVIEW`
 
 ---
 
 ### String filters:
 
-<a name="assigned"></a>
+<a name="reviewer"></a>
 
-#### assigned
+#### reviewer
 
-Example: `assigned=none`  
-Considered in implementation: **Yes** [#251 - Filters]  
-Can have combined values: **Yes** e.g: `assigned=none,"Carl"`  
+Included: **Yes**  
+Can have combined values: **Yes**  
+Examples: `reviewer=testReveiwer2` or `reviewer=testReveiwer2,testReviewer1`
+After is converted to Graphql: `reviewerUsernames: { overlaps: ["testReviewer2", "testReviewer1"]}`
 Options:
 
-- `none` - Applications with sections not-assigned
-- `"username"` - Reviewer username
+- `"username"` - Reviewer's username
 
-<a name="consolidator"></a>
+<a name="assigner"></a>
 
-#### consolidator
+#### assigner
 
-Example: `consolidator="Tony"`  
-Considered in implementation: **Yes** [#251 - Filters]  
-Can have combined values: **Yes** e.g: `consolidator="Tony","Andrei"`  
+Included: **Yes**  
+Can have combined values: **Yes**
+Examples: `assigner=testAssigner1` or `assigner=testAssigner1,Nicole`
+After is converted to Graphql: `assignerUsernames: { overlaps: ["testAssigner1", "Nicole"]}`
 Options:
 
-- `"username"` - Consolidator username
+- `"username"` - Assigner's username
 
 <a name="applicant"></a>
 
 #### applicant
 
-Example: `applicant="John"`  
-Considered in implementation: **No**  
-Can have combined values: **Yes** e.g: `applicant="John","Nicole"`  
+Included: **Yes**
+Can have combined values: **Yes**
+Exmples: `applicant=Carl` or `applicant=Carl Smith`
```

Comment:
Sp: "Examples"
-- also, are spaces allowed in this string?

Reply:
Thank you. Fixed now and replaced TODO as well.

Merging...